pkg/eventprocessor/eventfeed/impl/eventfeed.go

```diff
@@ -426,7 +427,11 @@ func (ef *EventFeed) persistEvents(ctx context.Context, events []types.Log, pars
 			ChainID:   ef.chainID,
 			EventJSON: eventJSONBytes,
 			EventType: eventType,
-		})
+		}
+		tblEvents = append(tblEvents, tblEvent)
+		if err := telemetry.Collect(ctx, toNewTablelandEvent(tblEvent)); err != nil {
+			return fmt.Errorf("collecting new tableland event metric: %s", err)
+		}
```

Comment:
Same logic as mentioned before. We first save as a metric, and then as an event.